kong/error.go

```diff
@@ -4,20 +4,24 @@ import (
 	"fmt"
 )
 
-type kongAPIError struct {
+type APIError struct {
```

Comment:
Can you add a doc line to this type and the functions?
This is required because this type is now exported.

Reply:
Sure, will add it.